### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by maksym-nemera**
[DEMO LINK](https://maksym-nemera.github.io/react_todo-app-add-and-delete/)

### CODE REVIEW

**Review by DidenkoDenis306 — APPROVED** [review]
GJ!

File: src/App.tsx [hunk 1]
```diff
@@ -1,24 +1,224 @@
...
+
+            <button
+              type="button"
+              className="todoapp__clear-completed"
+              disabled={!completedTodosIds.length}
+              onClick={() => {
+                removesTodo(completedTodosIds);
+                setLoadingTodos(completedTodosIds);
+                setstatusTodo(StatusValue.ALL);
+              }}
```
Comment (dffUqp) [inline src/App.tsx:210]: move it to a separate function

File: src/App.tsx [hunk 2]
```diff
@@ -1,24 +1,224 @@
...
+
+              <a
+                href="#/completed"
+                className={cn('filter__link', {
+                  selected: statusTodo === StatusValue.COMPLETED,
+                })}
+                onClick={() => setstatusTodo(StatusValue.COMPLETED)}
+              >
+                Completed
+              </a>
```
Comment (dffUqp) [inline src/App.tsx:199]: can we map our enum here?🤔

**Review by dffUqp — APPROVED** [review]
LGTM:fire:

**Pull request closed without merging**